### ISSUE TITLE
Improve Screen Reader support for Inner Toolbar shared buttons.

### ISSUE THREAD

**Issue by etoledom** [wordpress-mobile/gutenberg-mobile#911]
All blocks have the move Up/Down and the delete buttons.
These are not accessible.

They should read the state (if it's disabled), name, type.
